### DISCUSSION TITLE
Add accuracy to the graph

### DISCUSSION THREAD

**Discussion by vycdev** [monkeytypegame/monkeytype#1519] [Ideas]
It would be cool to have an option to enable accuracy on the graph at the end.

**Comment by frogfrogfrogfrogfrog** [comment]
By "enabling accuracy" do you mean turning off accuracy or are you implying that there isn't accuracy? If it's the former then I don't see the point in that because after a test you'd want to see all the information. If it's the latter then accuracy already exists in the graph when you finish a test.

**Reply by vycdev** [reply]
You get a number at the end, but it would be nice to see the evolution of your accuracy in the test, and yes, you already have errors showing on the graph, but I thought it would be cool if you had a line just like raw wpm or wpm that shows your accuracy.

**Reply by Miodec** [reply]
The problem with an accuracy line is that its not at all interesting. It would be mostly a straight line, and any major dips you can derive from errors that are already shown on the graph. I don't know.

It might also make the graph a bit busy.
